scripts/components/inline-notification/inline-notification.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { icons } from '@eightshift/frontend-libs/scripts/editor';
+import classnames from 'classnames';
+
+export const InlineNotificationType = {
+	warning: 'warning',
+	error: 'error',
+	info: 'info'
+}
+
+/**
+ * A simple inline notification to be used inside the Editor or Options.
+ * 
+ * @param {object} props                                                       - InlineNotification options.
+ * @param {string} props.text                                                  - Notification text.
+ * @param {InlineNotificationType} [props.type=InlineNotificationType.warning] - Help text displayed below the picker.
+ * @param {boolean} [props.showContrastOutline=false]                          - If `true`, a high-visibility outline is shown around the notification.
+ */
+export const InlineNotification = ({
+	text,
+	type = InlineNotificationType.warning,
+	showContrastOutline = false,
+}) => {
+	let icon = icons.infoCircle;
+
+	switch (type) {
+		case InlineNotificationType.error:
+			icon = icons.errorCircle;
+			break;
+		case InlineNotificationType.warning:
+			icon = icons.warning;
+			break;
+		default:
+			icon = icons.infoCircle;
+			break;
+	}
```

Comment:
you could maybe make another object something like

```suggestion
	type = InlineNotificationType.warning,
	showContrastOutline = false,
}) => {
	const notificationIcon = {
		warning: 'warning'
		error: 'errorCircle',
		somethingElse: 'redTurboError',
	}
```
and then call
```suggestion
	icon = notificationIcon[type] || notification.infoCircle;
```

Reply:
Made it a switch here in case I decide to add something in the future - with a switch it would be less hassle to add/change a new variable than with an object.